test/api/repositories/authentication_repository_test.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:flutter_config/flutter_config.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/mockito.dart';
+import 'package:survey_flutter/repositories/authentication_repository.dart';
+import 'package:survey_flutter/api/exception/network_exceptions.dart';
+import 'package:survey_flutter/model/response/login_response.dart';
+
+import '../../mocks/generate_mocks.mocks.dart';
+
+void main() {
+  group('AuthenticationRepositoryTest', () {
```

Comment:
Do we have a convention to name the `group` here, which the class name, or class name with the `Test` suffix will be used 🙏

Reply:
AFAIK, we don't have a convention to name the group, AuthenticationRepositoryTest is just a snake case of authentication_repository_test 🙏